Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/AuditRepositoryLogger.java

```diff
@@ -100,152 +103,179 @@
     private final AdminDistTransforms adAuditTransformer = new AdminDistTransforms();
     private final DocumentQueryTransforms dqAuditTransforms = new DocumentQueryTransforms();
     private NotifyTransforms transformLib = new NotifyTransforms();
+    private final COREX12RealtimeTransforms coreX12Realtime = new COREX12RealtimeTransforms();
```

Comment:
confusing variable name.  Can this be changed to something like: x12AuditTranforms